apps/camera/js/views/controls.js

```diff
@@ -251,10 +276,10 @@ module.exports = View.extend({
     '</div>' +
     '<div class="controls-right">' +
       '<div class="mode-switch test-switch" name="switch">' +
-        '<div class="inner js-switch">' +
-          '<div class="mode-switch_bg-icon rotates" data-icon="camera"></div>' +
-          '<div class="mode-switch_bg-icon rotates" data-icon="video"></div>' +
-          '<div class="mode-switch_handle js-switch-handle">' +
+        '<div class="inner js-switch" role="radiogroup">' +
+          '<div class="mode-switch_bg-icon js-radio-camera rotates" data-icon="camera" role="radio" data-l10n-id="camera-mode-radio-button"></div>' +
+          '<div class="mode-switch_bg-icon js-radio-video rotates" data-icon="video" role="radio" data-l10n-id="video-mode-radio-button"></div>' +
```

Comment:
Do we need to `data-l10n-id` attributes if we are manually setting `aria-label`? I'd like to remove any misleading code if possible, and just leave the functional parts.